tests/e2e/specs/admin-tracking.test.js

```diff
@@ -11,17 +6,17 @@ import { activatePlugin, deactivatePlugin, visitAdminPage } from '@wordpress/e2e
 /**
  * Internal dependencies
  */
-import { resetSiteKit } from '../utils';
+import { resetSiteKit, setSearchConsoleProperty } from '../utils';
 
-describe( 'Providing client configuration', () => {
+describe( 'management of tracking opt-in/out via settings page', () => {
 
 	beforeAll( async() => {
 		await activatePlugin( 'e2e-tests-auth-plugin' );
 		await activatePlugin( 'e2e-tests-site-verification-plugin' );
 	} );
 
 	beforeEach( async() => {
-		await resetSiteKit();
+		await setSearchConsoleProperty();
```

Comment:
Wouldn't it be better to have this work through a utility plugin? It's the same type of setup requirement like being authenticated and verified, which also have plugins.

Reply:
Perhaps - see my comment below.